src/main/jbake/content/blog/2017/07/grafana-new-query-interface.adoc

```diff
@@ -0,0 +1,174 @@
+= Grafana: new query interface
+Joel Takvorian
+2017-07-20
+:jbake-type: post
+:jbake-status: published
+:jbake-tags: blog, metrics, grafana
+:figure-caption!:
+
+There has been improvements lately in our Hawkular Grafana datasource, that are worth mentioning.
+
+- The way to query metrics by tags has changed since plugin link:https://github.com/hawkular/hawkular-grafana-datasource/releases/tag/v1.0.8[v1.0.8].
+It now takes advantage of the Hawkular Metrics' tags query language,
+that was introduced server-side in link:http://www.hawkular.org/blog/2017/02/08/hawkular-metrics-0.24.0.Final-released.html[Metrics 0.24.0]
+and enhanced in link:http://www.hawkular.org/blog/2017/03/07/hawkular-metrics-0.25.0.Final-released.html[0.25.0].
+To sum it up, Metrics integrates a parser that allows queries such as:
+`tag1 = 'awesome!' AND tag2 NOT IN ['foo', 'bar']`.
+
+- The datasource in now able to fetch bucketized metrics stats, instead of raw metrics.
+It consists in aggregating datapoints in slices of time (buckets) and providing, for each slice, some statistics like _min_, _max_, _average_ and more.
+The exact content of a bucket is described in link:http://www.hawkular.org/docs/rest/rest-metrics.html#NumericBucketPoint[Metrics REST API].
+Hawkular has always been able to provide them server-side,
```

Comment:
I mean the buckets (ie. stats). I could rephrase "Hawkular has always been able to provide metrics stats server-side"